beacon-chain/rpc/validator/attester.go

```diff
@@ -211,19 +217,22 @@ func (vs *Server) SubscribeCommitteeSubnets(ctx context.Context, req *ethpb.Comm
 		return uint64(len(vals)), nil
 	}
 
-	// run first request
+	// request the head validator indices of
```

Comment:
Why are the comments formatted like this?

Reply:
ok changed